eventuate-tram-consumer-rabbitmq/src/main/java/io/eventuate/tram/consumer/rabbitmq/Subscription.java

```diff
@@ -141,10 +144,12 @@ private void leaderSelected() {
   }
 
   private void leaderRemoved() {
+    leaderHook.ifPresent(hook -> hook.leaderUpdated(false, subscriptionId));
+
     logger.info("Revoking leadership from subscription. {}", identificationInformation());
     try {
       subscriberGroupChannel.close();
```

Comment:
Why close it here? Why not close it in leaderSelected()? Why does it need to be kept open?

Reply:
I will check